### DISCUSSION TITLE
Does iverilog run on Apple M1 macOS?

### DISCUSSION THREAD

**Discussion by balfieri** [steveicarus/iverilog#500] [Q&A] [answered]
Hello, I am hoping to get an M1 MBP soon. I love iverilog and use it a lot on my MBP for various projects.

Has someone confirmed that it builds and runs the tests on M1? I'm guessing it should be perfectly fine, but wanted to check.

Thanks.
Bob Alfieri
NVidia Research

**Comment by logicmonkey** [comment, marked as answer]
It builds and runs right out of the box on M1 (as does Verilator). I pull both from GitHub.
GTKWave is on Homebrew and is aarch64 native.

HTH
Piers Barber - Arm

**Reply by balfieri** [reply]
Cool. Thanks. I need to try Verilator.

Does GTKWave work on M1?

**Comment by logicmonkey** [comment]
Iverilog and Verilator are broken when using the version of bison that is part of Apple's Xcode.
To get them both working again I did a `brew install bison` and made sure it was in my path ahead for the Xcode one.
Bleeding edge Iverilog, Verilator and now SymbiYosis all good again for me.

**Reply by martinwhitaker** [reply]
Yes, this is already noted in the README (section 2.1). Of course it would be better if the configure script checked program versions...
